front50-web/src/main/groovy/com/netflix/spinnaker/front50/controllers/PipelineController.groovy

```diff
@@ -46,10 +46,10 @@ class PipelineController {
   @Autowired
   PipelineDAO pipelineDAO
 
-  @PreAuthorize("@fiatPermissionEvaluator.storeWholePermission()")
-  @PostFilter("hasPermission(filterObject.application, 'APPLICATION', 'READ')")
+  @PreAuthorize("#restricted ? @fiatPermissionEvaluator.storeWholePermission() : true")
+  @PostFilter("#restricted ? hasPermission(filterObject.name, 'APPLICATION', 'READ') : true")
   @RequestMapping(value = '', method = RequestMethod.GET)
-  List<Pipeline> list() {
+  List<Pipeline> list(@RequestParam(required = false, value = 'restricted', defaultValue = 'true') boolean restricted) {
```

Comment:
Either one would work. I opted for a query param for two reasons:

1.) the ?restricted=false appears in the logs by default
2.) consistent with the same thing listing Applications in Clouddriver.

I plan to submit as is, and can file a bug to move both servers and clients over to headers if you want.